extensions/amp-bento/0.1/amp-bento.css

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Special elements that require FOUC. */
+
+@import 'amp-accordion.css';
+@import 'amp-social-share.css';
+
+
+/* Common boilerplate */
+
+/* display:block elements */
+amp-accordion,
+amp-base-carousel,
+amp-fit-text,
+amp-inline-gallery,
+amp-inline-gallery-pagination,
+amp-inline-gallery-thumbnails,
+amp-instagram,
+amp-selector,
+amp-stream-gallery {
```

Comment:
I'm not super happy about this enumeration, it feels just like never ending boilerplate that we would have to include on pages.

What if we required each extension to provide these styles? And have pubs include both `amp-video-0.1.js` and `amp-video-0.1.css` on their page? We could optimize that during serving into a single `<style bento-runtime>` inlined on the page.

Reply:
Could we have both? Include component css if using < n components, include this if you're using many, similar to our experimental flag?